python_dwd/file_path_handling/file_index_creation.py

```diff
@@ -0,0 +1,119 @@
+""" file index creation for available DWD station data """
+from pathlib import Path, PurePosixPath
+import re
+from typing import Union
+from tqdm import tqdm
```

Comment:
Let's add this module also to `requirements.txt`, right?

Reply:
I removed the functionality, should be fine without it.